.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "0.10.40"
+  - "4"
```

Comment:
@sculove read the doc
https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Available-Versions